protocol/v2/ssv/runner/runner.go

```diff
@@ -179,6 +179,7 @@ func (b *BaseRunner) baseConsensusMsgProcessing(logger *zap.Logger, runner Runne
 	}
 
 	decidedMsg, err := b.QBFTController.ProcessMsg(logger, msg)
+	// TODO: Should we remove this function since it doesn't do anything?
```

Comment:
@y0sher Can I remove this `compactInstanceIfNeeded` function?

Reply:
@MatheusFranco99 let's remove it as it's empty right now

Reply:
Removed